main.go

```diff
@@ -452,6 +453,41 @@ func main() {
 			}
 		})
 
+		http.HandleFunc("/mount-health-check", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
`mount-health-check` is too specific given you are actually "only" checking something in the file system no matter whether how it is mounted. What about those more generic operations to get rid of "mount" ?

* `HEAD /filesystem?path=<path`, returns 200 if path exists and is file, 204 if path exists but is directory, 404 if not found, 500 if other error
* `GET /filesystem?path=<path>`, returns 200 if path exists and is file and returns content of file in response body, 204 if path exists but is directory, 404 if not found, 500 if other error

Make sure to also update README.